obal/data/modules/check_koji_build.py

```diff
@@ -23,23 +23,35 @@ def main():
     package = module.params['package']
     koji_executable = module.params['koji_executable']
 
+    command = ['buildinfo', nevr]
     try:
-        if tag:
-            command = ['latest-build', '--quiet', tag, package]
-            build = koji(command, koji_executable)
-            build = build.split(' ')[0]
+        output = koji(command, koji_executable)
+    except KojiCommandError as error:
+        module.fail_json(changed=False, msg=error.message, command=error.command)
```

Comment:
when the build doesn't exist, `koji buildinfo` exits non-zero, which will trigger this Exception, even if you don't want it:

```console
% koji buildinfo hello-123-1.el7; echo $?       
No such build: hello-123-1.el7

1
```